src/components/Footer/index.tsx

```diff
@@ -48,7 +48,9 @@ export default function () {
               content={FooterOptions[key].text}
             />
             {linkList.length - 1 !== index && (
-              <Delimiter color="dimmed" showFrom="md" />
+              <div className={displayFrom('md')}>
+                <Delimiter />
```

Comment:
imo dark blue-purple color has bad contrast:
![image](https://user-images.githubusercontent.com/49658988/194407267-1aa42843-e2c9-4b05-87ad-d6ccd03ba54f.png)

Reply:
It looks like in the design. We may ask Matt on Monday about that